addon/components/o-s-s/array-input.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="array-input-container fx-row fx-xalign-center" ...attributes>
+<div class="array-input-container fx-row fx-xalign-center fx-gap-px-6 padding-left-px-6 padding-right-px-6" ...attributes>
```

Comment:
80% chance where we use array-input will look weird after we merge this in.
I'm pretty sure I adjusted some css where we used some of these components... Nothing too awful though, we can fix them when we see them 🤷

Reply:
I double checked in the CCs and BCCs in the composer and it works well w/ the adjustments done on the Chip too.